app/react/components/tweet-column.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class TweetColumn extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { tweets: '' }
+  }
+
+  componentDidMount() {
+    $.getJSON('/tweets', (response) => {
```

Comment:
Don't use jQuerry if you don't have to. Check out `fetch`: https://developer.mozilla.org/en/docs/Web/API/Fetch_API

Also, if you can save 'response' variable without any serialisation as 'tweets' you may reconsider naming it 'response' in the first place.